### PR TITLE
Implement vitals dashboard responsive design

### LINKED ISSUE #975: Update responsive design for vitals dashboard
**What needs to be done? Why does it need to be done?**
Update the design for the Vitals dashboard to be responsive to tablet screen widths. 
image (1)
hannel=C018TBEM48G

**Additional context**
Mocks: https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=2022%3A3

### PR DESCRIPTION
**Pull request by daschi**
## Description of the change

Adds styles for breakpoints at 1366, 1024, 768 for tablet views. 

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #975

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/VitalsSummaryTable/VitalsSummaryTable.scss [hunk 1]
```diff
@@ -67,19 +99,36 @@ $border-styles: 1px solid $slate-20;
...
         font-weight: 500;
+        width: 175px;
+
+        > .VitalsSummaryTable__sortable-header {
+          @media screen and (max-width: $core-breakpoint-md) {
+            width: 122px;
+          }
+        }
+
+        &#entity {
```
Comment (jovergaag) [inline src/core/VitalsSummaryTable/VitalsSummaryTable.scss:110]: This isn't working in IE11

File: src/core/PageVitals/PageVitals.scss [hunk 2]
```diff
@@ -23,9 +23,52 @@
     justify-content: space-between;
     width: 100%;
 
-    :not(:last-child) {
+    > :not(:last-child) {
       margin-right: 1.5rem;
     }
+
+    @media screen and (max-width: $core-breakpoint-sm) {
```
Comment (jovergaag) [inline src/core/PageVitals/PageVitals.scss:30]: I tested IE11 at 1000px screen width (not a likely scenario but doesn't hurt to fix it). The Summary cards are not wrapping.

Screen Shot 2021-04-01 at 3 51 14 PM

**Review by jovergaag — CHANGES_REQUESTED** [review]
Beautiful! IE11 is of course...the problem child. 😈

File: src/core/PageVitals/PageVitals.scss (hunk 2 above)
Comment (daschi) [reply src/core/PageVitals/PageVitals.scss:30]: Hm yeah I chatted with Sonia about this and she said it wasn't P0 to fix for IE11, but I'll spend a bit of time this morning to see if it's an easy fix

File: src/core/VitalsSummaryTable/VitalsSummaryTable.scss (hunk 1 above)
Comment (daschi) [reply src/core/VitalsSummaryTable/VitalsSummaryTable.scss:110]: Yeah, `position: sticky` isn't supported in IE11, I think we should either not have the fixed column be a feature in IE11, or make a separate issue to investigate the best way to implement this in a table for IE11.

Comment (jovergaag) [reply src/core/VitalsSummaryTable/VitalsSummaryTable.scss:110]: I made an [issue](https://app.zenhub.com/workspaces/torch--5ec36552cdfe7bc58f04bdbe/issues/recidiviz/pulse-dashboard/992) for this for next sprint.

**Review by jovergaag — APPROVED** [review]
🎨

**Pull request merged**